From e46fb585954a131a8f96cfd71cf2705fe0ff1e02 Mon Sep 17 00:00:00 2001
From: NitinKM <70827815+NewtonChutney@users.noreply.github.com>
Date: Mon, 8 Jan 2024 08:16:53 +0530
Subject: [PATCH 172/188] Info on how to skip grammar build when building from
 source (#8698)

* info: no grammar compile

Added instructions on how to compile without compiling grammars

* Update book/src/install.md

Co-authored-by: Michael Davis <mcarsondavis@gmail.com>

---------

Co-authored-by: Michael Davis <mcarsondavis@gmail.com>
---
 book/src/install.md | 2 ++
 1 file changed, 2 insertions(+)

--- a/book/src/install.md
+++ b/book/src/install.md
@@ -204,6 +204,8 @@ RUSTFLAGS="-C target-feature=-crt-static
    This command will create the `hx` executable and construct the tree-sitter
    grammars in the local `runtime` folder.
 
+> ðŸ’¡ If you do not want to fetch or build grammars, set an environment variable `HELIX_DISABLE_AUTO_GRAMMAR_BUILD`
+
 > ðŸ’¡ Tree-sitter grammars can be fetched and compiled if not pre-packaged. Fetch
 > grammars with `hx --grammar fetch` and compile them with
 > `hx --grammar build`. This will install them in
